src/internal/storage/fileset/storage.go

```diff
@@ -465,3 +465,17 @@ func (d *deleter) DeleteTx(tx *pachsql.Tx, oid string) error {
 	}
 	return d.store.DeleteTx(tx, *id)
 }
+
+type PinnedFileset ID
```

Comment:
Is this type going to add value?

Reply:
Yeah it will be used in the PJS function signatures to add some type safety and force a caller to pin their filesets before they create a job.